js/controller.js

```diff
@@ -32,6 +32,7 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
     this.videoVrSource = null;
     this.videoVr = false;
     this.captionDirection = '';
+    this.isNewVideo = true;
```

Comment:
This is only ever set to true.  Do we need this? 

In general we keep track if the video is a replay or not. Not whether it's a new video. To keep it consistent with all the other codes in all the repos I suggest changing this.

Reply:
change to isNewVrVideo

Reply:
\+